ydb/core/kqp/provider/yql_kikimr_gateway.cpp

```diff
@@ -120,6 +120,7 @@ void IKikimrGateway::BuildIndexMetadata(TTableMetadataResult& loadTableMetadataR
 
         TString error;
         auto indexType = TIndexDescription::ConvertIndexType(index.Type);
+        YQL_ENSURE(indexType != NKikimrSchemeOp::EIndexType::EIndexTypeGlobalVectorKmeansTree);
```

Comment:
I have taken this line to another PR https://github.com/ydb-platform/ydb/pull/6602

```suggestion
```

Reply:
Ok